src/Scanner.js

```diff
@@ -170,6 +170,15 @@ class DefaultScanner {
                 })
             }
         }
+
+        // Check for forms with too few 'known' inputs
+        // In the case where the only inputs are all 'unknown', we can destroy the form and stop tracking it.
+        for (let formInstance of this.forms.values()) {
+            if (formInstance.hasOnlyUnknownFields) {
+                formInstance.destroy()
+            }
+        }
+
```

Comment:
This is done *after* adding all inputs, because only then can you determine whether it's truly useless or not.